tests/e2e.test.js

```diff
@@ -295,17 +296,34 @@ describe('Ordering', () => {
     }
   });
 
-  it.skip('orders grid items by color', async () => {
-    // Items are ordered using the CSS property `order`...
-    await expect(page).toClick('#order-color');
+  it('orders grid items by color', async () => {
+    const $items = await page.$$('li.grid-item');
+    for (let i = 0; i < $items.length; i++) {
+      const $button = await $items[i].$('button.grid-item__color');
 
-    const $gridItems = await page.$$('.copy-color');
-    for (let i = 0; i < $gridItems.length; i++) {
-      const $gridItem = $gridItems[i];
-      const hex = hexes[i];
-      await expect($gridItem).toMatch(`#${hex}`);
+      const $text = await $items[i].evaluate((el) =>
+        JSON.stringify(el.getAttribute('style')),
+      );
+      const $idx = parseInt($text.substring(15, $text.length - 1));
```

Comment:
You can use the [`getAttribute` helper](https://github.com/simple-icons/simple-icons-website/blob/00139387a7133ce1eadd86eb70a0e28e8f35dac4/tests/helpers.js#L14-L20) here.

```suggestion
      const $text = await getAttribute($items[i], 'style');
      const $idx = parseInt($text.substring(14, $text.length));
```

Also, I don't understand why you're `JSON.stringify`ing the result of `getAttribute()` here.